src/libponyc/codegen/genjit.cc

```diff
@@ -280,12 +280,12 @@ bool gen_jit_and_run(compile_t* c, int* exit_code, jit_symbol_t* symbols,
 
```

Comment:
Changes in genjit.cc aren't related to this RFC. Can you revert the changes to this file?